src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -1900,6 +1900,13 @@ def test_has_read_agreement(self):
             response, reverse('devhub.submit.version.agreement', args=[self.addon.slug])
         )
 
+    def test_site_permission_not_allowed(self):
+        self.addon.update(type=amo.ADDON_SITE_PERMISSION)
+        response = self.client.get(self.url)
+        assert response.status_code == 403
+        response = self.client.post(self.url)
+        assert response.status_code == 403
```

Comment:
nit: a mixin for this?  Seems a bit un-DRY-y to have the same chunk of code 4 times.